tests/ui/join_absolute_path.rs

```diff
@@ -0,0 +1,26 @@
+//@run-rustfix
```

Comment:
rustfix only works with valid suggestions, for now, the best action is to rust remove it :)
```suggestion
```